src/pages/Checklist/index.tsx

```diff
@@ -168,78 +177,186 @@ export const Checklist = () => {
     <PageTemplate title="Eth2 validator checklist">
       <Subtitle>
         <FormattedMessage defaultMessage="This checklist will help you understand the role of a validator and prepare you for the role." />
+        <Text className="mt10">
+          <FormattedMessage
+            defaultMessage="Visit EthStaker on {discord} or {reddit} at any time during your onboarding for some friendly help!"
+            values={{
+              discord: (
+                <Link primary inline to="https://invite.gg/ethstaker">
+                  Discord
+                </Link>
+              ),
+              reddit: (
+                <Link primary inline to="https://reddit.com/r/ethstaker">
+                  Reddit
+                </Link>
+              ),
+            }}
+            description="{variables} are social media platform links to Discord and Reddit (do not translate names)"
+          />
+        </Text>
       </Subtitle>
       <ChecklistPageStyles>
+        <SectionHeader>
+          <Heading level={3}>
+            <FormattedMessage defaultMessage="Section 1 - Preparation" />
+          </Heading>
+        </SectionHeader>
         <section>
-          <Heading level={3}>Hardware</Heading>
-          <Heading level={4}>Recommended</Heading>
+          <Heading level={4}>
+            <FormattedMessage defaultMessage="Recommendation disclaimer" />
+          </Heading>
           <ul className="sub-checklist-item">
             <li className="py5">
               <Text>
                 <FormattedMessage
-                  defaultMessage="In order to process incoming validator deposits from the eth1
-                    chain, you will need to run an eth1 client in parallel to your
+                  defaultMessage="Please note that hardware suggestions are an ever evolving target. Current
+                    minimum requirements are likely to increase by an order of magnitude after the merge and 
+                    introduction of shard chains. Please do your own research before depositing funds."
+                />
+              </Text>
+            </li>
+          </ul>
+        </section>
+        <section>
+          <Heading level={3}>
```

Comment:
Link to this from the new homepage "Become a validator" steps?

Reply:
Thinking here was "Become a validator" takes you into the disclaimer overview, of which this checklist is now one of the items presented to new users. Meanwhile the "Thinking about staking?" button leads to the checklist.

Makes me think linking to the FAQ from the end of the checklist may not be a bad idea